app/models/manageiq/providers/kubernetes/container_manager/refresh_parser.rb

```diff
@@ -772,7 +768,7 @@ def parse_image_name(image, image_ref)
         {
           :name          => image_parts[:name],
           :tag           => image_parts[:tag],
-          :digest        => image_parts[:digest] || (image_ref_parts[:digest] if image_ref_parts),
+          :digest        => image_parts[:digest] || (image_ref_parts[:digest] if image_ref_parts) || "",
```

Comment:
Just curious, why did you want to change the empty value from nil to empty string?
(given that ```#{nil}``` is an empty string)